app/forms/validators.py

```diff
@@ -0,0 +1,8 @@
+from datetime import date
+
+from wtforms.validators import ValidationError
+
+def is_not_past(form, field):
+    print("validuje date")
```

Comment:
printa lepiej na loggera zamienić

Reply:
To było do debugowania zapomniałem usunąć xD sorry

Reply:
Luzik, wypisuje po prostu wszystko co mi się rzuca w oczy ;)